src/verify-app-guide/verify-app-guide.component.js

```diff
@@ -0,0 +1,157 @@
+            </section>
+          </>
+        )}
+        <section>
+          Got stuck? you can always hit us in our{" "}
+          <SlackLink>single-spa slack workspace</SlackLink> or{" "}
+          <GithubPlaygroundIssuesLink>
+            file a github issue
+          </GithubPlaygroundIssuesLink>
+        </section>
+
+        <section>
+          <div style={{ margin: "16px 0" }}>
+            {!application && (
+              <button
+                className="secondary"
+                type="button"
+                onClick={() => setShowRegisterApp(true)}
+              >
+                {`Register app`}
+              </button>
+            )}
+          </div>
+        </section>
+        {showRegisterApp && (
+          <EditRegisteredApp
+            addApp={addAndClose}
+            cancel={() => setShowRegisterApp(false)}
+            updateApp={updateApp}
+          />
+        )}
+
+        {application && (
+          <>
+            <section>
+              Now, lets verify if your application is ready to go! If you came
+              from create-single-spa, this should work out-of-the-box, but you
+              can always come back after making changes to your single-spa
+              application to check if it still conforms to single-spa API
+            </section>
+            <section>
+              <button
+                className="primary"
+                type="button"
+                onClick={() => {
+                  setShowVerificationSteps(true);
+                  setShowRegisterApp(false);
+                }}
+              >
+                {`Verify "${application.name}"`}
+              </button>
+            </section>
+
+            {showVerificationSteps && <VerificationSteps app={application} />}
+
+            <section>
+              Once you've verified things are working as expected, proceed to
+              the next step.
+            </section>
+            <section className="actions next-step">
+              <Link to="/root-config-guide" className="primary button">
+                Next step: root-config
+              </Link>
+            </section>
+          </>
+        )}
+      </article>
+    </>
+  );
+
+  function edit(app) {
+    setIsEditingApp(true);
+  }
+
+  function updateApp(app, url, oldName) {
+    updateApplication(app);
+    const oldUrl = window.importMapOverrides.getOverrideMap().imports[app.name];
+    if (oldUrl !== url) {
+      window.importMapOverrides.removeOverride(oldName);
+      window.importMapOverrides.addOverride(app.name, url);
+      // Changes to the import map require a page reload
+      window.location.reload();
+    }
+    setIsEditingApp(false);
+  }
+
+  function addAndClose(app, url) {
+    window.importMapOverrides.addOverride(app.name, url);
+    app.url = url;
+    addApplication(app);
+    window.location.reload();
+    setShowRegisterApp(false);
+  }
+}
+
+const css = `
+  &
```

Comment:
Let's just remove the entire `css` variable since it's unused

Reply:
oops